cg/meta/orders/schema.py

```diff
@@ -110,22 +110,22 @@ def validate(self, value):
 
     # This information is required
     'name': validators.RegexValidator(NAME_PATTERN),
-    'container': validators.Any(CONTAINER_OPTIONS),
+    'container': validators.Optional(TypeValidator(str, allow_none=True), None),
     'data_analysis': str,
     'application': str,
     'sex': validators.Any(SEX_OPTIONS),
     'family_name': validators.RegexValidator(NAME_PATTERN),
     'require_qcok': bool,
     'tumour': bool,
-    'source': str,
+    'source': validators.Optional(TypeValidator(str, allow_none=True), None),
     'priority': validators.Any(PRIORITY_OPTIONS),
 
     # Required if Plate
     'container_name': validators.Optional(str, None),
     'well_position': validators.Optional(TypeValidator(str, allow_none=True), None),
 
     # This information is required for Balsamic analysis (cancer)
-    'capture_kit': validators.Any(CAPTUREKIT_CANCER_OPTIONS),
+    'capture_kit': validators.Optional(TypeValidator(str, allow_none=True), None),
```

Comment:
hehe, it's not required when it's optional ;) Don't have a good suggestion on how to solve this tho